gocron_test.go

```diff
@@ -90,36 +90,15 @@ func TestScheduler_WeekdaysTodayAfter(t *testing.T) {
 	}
 }
 
-func TestScheduleNextRunLoc(t *testing.T) {
```

Comment:
this test didn't make sense anymore since we don't rely on jobs timezone anymore